bsp/stm32/stm32l4r5-st-nucleo/README.md

```diff
@@ -0,0 +1,116 @@
+﻿# STM32L4R5ZI-NUCLEO 开发板bsp说明
+
+## 简介
+
+由 QYZ 为 STM32L4R5ZI-NUCLEO 开发板提供的 BSP (板级支持包) 说明。
+
+主要内容如下：
+
+- 开发板资源介绍
+- BSP 快速上手
+- 进阶使用方法
+
+通过阅读快速上手章节开发者可以快速地上手该 BSP，将 RT-Thread 运行在开发板上。在进阶使用指南章节，将会介绍更多高级功能，帮助开发者利用 RT-Thread 驱动更多板载资源。
+
+## 开发板介绍
+
+开发板外观如下图所示：
+
+![board](figures/board.bmp)
```

Comment:
Please use jpeg image to reduce file size.

Reply:
ok

Reply:
ok